core/plugins/org.polarsys.capella.core.diagram.helpers/META-INF/MANIFEST.MF

```diff
@@ -21,7 +21,8 @@ Require-Bundle: org.polarsys.capella.core.model.helpers;visibility:=reexport,
  org.eclipse.emf.edit,
  org.eclipse.emf.common,
  org.eclipse.sirius.diagram.sequence,
- org.eclipse.sirius.ui
+ org.eclipse.sirius.ui,
+ org.polarsys.capella.core.platform.sirius.clipboard
```

Comment:
Inverse dependency has been added in other pull request https://github.com/eclipse/capella/pull/238/files#diff-68fb93a3758604b2f3627e5202a275b5R19

Reply:
Nice catch.